full_snapshot.proto

```diff
@@ -400,8 +400,8 @@ message RelationInformation {
   bool has_inheritance_children = 9;
   bool has_toast = 10;
 
-  uint32 frozen_xid = 11;
-  uint32 minimum_multixact_xid = 12;
+  uint32 frozen_xid = 11; // deprecated in favor of relfrozenxid in RelationStatistic
+  uint32 minimum_multixact_xid = 12; // deprecated in favor of relminmxid in RelationStatistic
```

Comment:
Should we mark these as `reserved`? (i.e. `reserved 11` and `reserved 12`?)

We can leave the comments in place, but that makes it clear we don't expect users of the updated protobuf definitions to work with these fields anymore.

Reply:
Sure. I just did what we did for `domain_constraint`, but `git blame` points the finger at me. I'll update these and that one to be reserved, like other deprecated fields.

Reply:
Sounds good - its not a strong preference from my side, but feels a bit easier when reading the protobuf definition, since you can easily just read past lines starting with "reserved".